Schemas/ploddSchemaSpecification.odd

```diff
@@ -375,6 +375,7 @@
             <attDef ident="rend"      mode="delete"/>
             <attDef ident="style"     mode="delete"/>
             <attDef ident="rendition" mode="delete"/>
+            <attDef ident="expand"    mode="delete"/>
```

Comment:
TEI has not committed to dropping this attr on https://github.com/TEIC/TEI/issues/2369. That said, a) they probably will; and b) until they either drop it or clearly define its semantics, there is no need for us to fret over how to process it. So, at least for now (and probably permanently) it makes sense to me to disallow it.